content/en/integrations/guide/aws-organizations-setup.md

```diff
@@ -0,0 +1,109 @@
+
+The Datadog CloudFormation StackSet performs the following steps:
+
+1. Deploys the Datadog AWS CloudFormation Stack in every account under an AWS Organization or Organizational Unit.
+2. Automatically creates the necessary IAM role and policies in the target accounts.
+3. Automatically initiates ingestion of AWS CloudWatch metrics and events from the AWS resources in the accounts.
+4. Optionally disables metric collection for the AWS infrastructure. This is useful for Cloud Cost Management (CCM) or Cloud Security Posture Management (CSPM) specific use cases.
+5. Optionally configures Datadog Cloud Security Posture Management to monitor resource misconfigurations in your AWS accounts.
+
+**Note**: The StackSet does not set up log forwarding in the AWS accounts. To set up logs, follow the steps in the [Log Collection][2] guide.
+
+
+## Prerequisites
+
+1. **Access to the management account**: Your AWS user needs to be able to access the AWS management account.
+2. **An account administrator has enabled Trusted Access with AWS Organizations**: Refer to [Enable trusted access with AWS Organizations][3] to enable trusted access between StackSets and Organizations, to create and deploy stacks using service-managed permissions.
+
+## Setup
+
+To get started, go to the [AWS Integration configuration page][1] in Datadog and click on **Add AWS Account(s)** -> **Add Multiple AWS Accounts** -> **CloudFormation StackSet**.
+
+Click **Launch CloudFormation StackSet**. This opens the AWS Console and loads a new CloudFormation StackSet. Keep the default choice of `Service-managed permissions` on AWS.  
+  
+Follow the steps below on the AWS console to create and deploy your StackSet:
+
+1. **Choose a Template**  
+Copy the Template URL from the Datadog AWS integration configuration page to use in the `Specify Template` parameter in the StackSet.
+
+
+2. **Specify StackSet details**
+    - Select your Datadog API key on Datadog AWS integration configuration page and use it in the `DatadogApiKey` parameter in the StackSet.
+    - Select your Datadog APP key on Datadog AWS integration configuration page and use it in the `DatadogAppKey` parameter in the StackSet.
+
+    - *Optionally:*  
+        a. Enable [Cloud Security Posture Management][5] (CSPM) to scan your cloud environment, hosts, and containers for misconfigurations and security risks.  
+        b. Disable metric collection if you do not want to monitor your AWS infrastructure. This is recommended only for [Cloud Cost Management][6] (CCM) or [Cloud Security Posture Management][5] (CSPM) specific use cases.
```

Comment:
```suggestion
        b. Disable metric collection if you do not want to monitor your AWS infrastructure. This is recommended only for [Cloud Cost Management][6] (CCM) or [CSPM][5] specific use cases.
```

Reply:
Just looking at this again, we can probably shorten `b` since that initialism is already expanded in `a`.